rllib/algorithms/ppo/ppo.py

```diff
@@ -359,6 +366,13 @@ def validate(self) -> None:
         if isinstance(self.entropy_coeff, float) and self.entropy_coeff < 0.0:
             raise ValueError("`entropy_coeff` must be >= 0.0")
 
+    @property
```

Comment:
Let's not support this mode for PPO. This was a very DreamerV3 specific thing to do. For PPO, you 'd usually run with lots of rollout workers, so this would be the absolute exception not worth creating this extra logic for.